about.md

```diff
@@ -4,10 +4,15 @@ title: About
 permalink: /about/
 navigation_order: 6
 ---
-
 Monthly Music Hackathon NYC is a monthly all-day event for musicians and engineers to create new music-related projects from scratch, develop them throughout ~10 hours, then perform or present them in a concert in the evening.
 
+***
```

Comment:
Hmmm it seems to me that the only thing needed is:

````
# FAQ
{:.no_toc}

{:toc}
* toc
````

Did that not work for you?